contracts/SingleBatchEdition.sol

```diff
@@ -120,12 +118,20 @@ contract SingleBatchEdition is
                    COLLECTOR / TOKEN OWNER FUNCTIONS
     //////////////////////////////////////////////////////////////*/
 
+    function revertIfNotAuthorizedMinter() internal view {
+        State memory _state = state;
```

Comment:
why not just use `state` directly?

Reply:
it used to make sense to cache the variable, now you're right we could use it directly because it is used only once